src/core/src/pass/manager.cpp

```diff
@@ -138,9 +140,13 @@ void ov::pass::Manager::run_passes(shared_ptr<ov::Model> func) {
         if (profile_enabled) {
             cout << setw(7) << pass_timer.get_milliseconds() << "ms " << pass->get_name() << "\n";
         }
+        function_changed = function_changed || pass_applied;
+        needs_validate = needs_validate || pass_applied;
```

Comment:
looks correct, but if there is a mistake, it will significantly affect performance. Could you double-check perfomance for couple of models (locally or via dl bench)

Reply:
tests shared offline